docs/NNNN-direct-clean-path-layout.md

```diff
@@ -0,0 +1,255 @@
+# OCFL Community Extension NNNN: Direct Clean Path Layout
+
+* **Extension Name:** NNNN-direct-clean-path-layout
+* **Authors:** Jürgen Enge, Basel
+* **Minimum OCFL Version:** 1.0
+* **OCFL Community Extensions Version:** 1.0
+* **Obsoletes:** n/a
+* **Obsoleted by:** n/a
+
+## Overview
+
+This extension can be used as root storage layout extension to map OCFL object
+identifiers to path names as well as a mapping for content path names.
+This is done by replacing or removing "dangerous characters" from names.
+
+The functionality has been derived from David A. Wheeler - "Fixing Unix/Linux/POSIX Filenames" (https://www.dwheeler.com/essays/fixing-unix-linux-filenames.html)
+
+#### Usage Scenario
+
+If you want to make sure, that the internal layout of the OCFL content 
```

Comment:
This might read better as something along the lines of:

> This extension is intended to be used when you want the internal OCFL layout to match the logical structure of the stored objects as closely as possible, while also ensuring all file and directory names do not include problematic characters.